mm2src/coins/eth/tron.rs

```diff
@@ -13,12 +19,27 @@ pub enum Network {
     // TODO: Add more networks as needed.
 }
 
-/// Placeholder for a TRON client.
+/// Draft TRON clients structure.
 #[derive(Clone, Debug)]
-pub struct TronClient;
+pub struct TronClients {
+    pub clients: Vec<TronClient>,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct TronClient {
+    pub endpoint: String,
+    pub network: Network,
+    #[serde(default)]
+    pub komodo_proxy: bool, // should be true for any net which requires api key
+}
 
 /// Placeholder for TRON fee params.
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct TronFeeParams {
     // TODO: Add TRON-specific fields in future steps.
 }
+
+#[allow(dead_code)]
```

Comment:
If you do it this way we can remember to remove it when it's used.

```suggestion
#[expect(dead_code)]
```

Reply:
you are right, removing pub solved the issue https://github.com/KomodoPlatform/komodo-defi-framework/pull/2467/commits/3d638415c49dccad0a8fec31778a834614f71ebd